source/process/release-process.md

```diff
@@ -229,20 +211,19 @@ The final release is cut. If an urgent and important issue needs to be addressed
       - Test the download links before and after updating the page
     - Update [MVP page](https://www.mattermost.org/mvp/) with the most valuable contributor of the release
```

Comment:
Line 205 (can't comment there) - should we be running upgrade tests so late (does having them here mean any upgrade issues will result in a dot release?) Do we ever hit problems with that?

Reply:
I think it would be better to move it earlier. If there's an issue we'd need to cut a dot release for it.

Reply:
Do I confirm with Christopher?

Reply:
@amyblais Perhaps post in a public channel and get developers thoughts on it. You can tag Christopher, but it doesn't hurt to get others' thoughts too

Reply:
Note from Chris: "I'd vote to keep the manual tests late, but maybe prioritize adding automated tests [daily] if we're having issues catching upgrade bugs early."

Reply:
I added some daily automated tests, can be edited.